src/persist/helper.ts

```diff
@@ -1,13 +1,23 @@
 import { Model } from '../model';
+import * as parse from 'csv-parse/lib/sync';
 
 export class Helper {
   public static loadPolicyLine(line: string, model: Model): void {
-    if (!line || line.trim() === '' || line.charAt(0) === '#') {
+    if (!line || line.charAt(0) === '#') {
```

Comment:
Maybe exist something like `[blank space]# This is a comment`

Reply:
good point.